dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -49,14 +49,14 @@ const findAllAmendmentsByFacilityId = async (facilityId) => {
   try {
     const collection = await db.getCollection('tfm-facilities');
     const amendment = await collection
-      .aggregate([ // TODO SR-8
-        { $match: { _id: ObjectId(facilityId) } },
-        { $project: { _id: 0, amendments: '$amendments' } },
+      .aggregate([
+        { $match: { _id: { $eq: ObjectId(facilityId) } } },
+        { $project: { _id: false, amendments: '$amendments' } }, // TODO SR-8 Changed values in project to true or false to better represent the functionality
         { $unwind: '$amendments' },
-        { $sort: { 'amendments.version': -1 } },
+        { $sort: { 'amendments.version': -1 } }, // TODO SR-8: Sort order -- no $eq expression needed
         { $match: { 'amendments.status': { $ne: CONSTANTS.AMENDMENT.AMENDMENT_STATUS.NOT_STARTED } } },
         { $group: { _id: '$_id', amendments: { $push: '$amendments' } } },
-        { $project: { _id: 0, amendments: 1 } },
+        { $project: { _id: false, amendments: true } }, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
        { $project: { _id: false, amendments: true } },
```